scripts/travis.sh

```diff
@@ -1,60 +1,27 @@
 #!/bin/bash
 
-SUBFOLDERS=("docs")
-SUBFOLDER=$1
-
-function test {
-    set -ev
-    npm install
-    npm run build
-    cd $TRAVIS_BUILD_DIR/$SUBFOLDER
-    npm install
-    
-    if npm run | grep test:travis > /dev/null
-    then
-        npm run test:travis
-    else
-        npm test
-    fi
-    exit
-}
-
-if [ "$TRAVIS_PULL_REQUEST" != "false" ]
-then
-    TRAVIS_COMMIT_RANGE="FETCH_HEAD..$TRAVIS_BRANCH"
-fi
-
-CHANGES=$(git --no-pager diff --name-only $TRAVIS_COMMIT_RANGE)
-
-# If .travis.yml or scripts/travis.sh changed, run all tests
-if echo $"$CHANGES" | grep -qE "^(\.travis\.yml|scripts\/travis\.sh)$"
+# Build sources
+if [[ "${1}" == "build" ]]
 then
-    test
+    set -ev
+    yarn build
+    set +ev
 fi
 
-# If no subfolder, check if anything else than a subfolder has changed
-if [ -z "$1" ]
-then
-    for DIR in ${SUBFOLDERS[@]}
-    do
-        CHANGES=$(echo "$CHANGES" | grep -v "^$DIR/")
-    done
-
-    if [ ! -z "$CHANGES" ]
-    then
-        test
-    fi
-
-# If there is a subfolder, only run tests if said folder has changes
-elif echo "$CHANGES" | grep -q "^$SUBFOLDER/"
+# Lint sources
+if [[ "${1}" == "lint" ]]
 then
-    test
+    set -ev
+    yarn lint
+    set +ev
 fi
 
-# always run tests for docs
-if [ $1 == "docs" ]
+# Tests for docs
+if [[ "${1}" == "docs" ]]
 then
-    test
+    set -ev
+    yarn build
+    cd packages/docs
+    yarn test
```

Comment:
Looks like you should add something like this back:
```
    if npm run | grep test:travis > /dev/null
    then
        npm run test:travis
    else
        npm test
    fi
```

Basically you're not even calling the @artipost thing right now 😉

Reply:
Fixed

> (no "archive is done", "uploading" and all that)

The doc build crashes before anyway right now